chat.html

```diff
@@ -6,6 +6,12 @@
     <script src="https://media.twiliocdn.com/sdk/js/flex-webchat/releases/2.1.2/twilio-flex-webchat.min.js"></script>
     <script>
       const translations = {
+        'ar': {
+          MessageCanvasTrayContent: "<p>المستشار غادر الدردشة. شكرا لك على التواصل. يرجى الاتصال بنا مرة أخرى إذا كنت بحاجة إلى مزيد من المساعدة.</p>"
```

Comment:
Arabic is a right-to-left language.  Please change the `<p>` tag to `<p dir='rtl'>` and check that it looks good and the periods appear at the end (which would be at the left of the word).  I think this is what we need:

![image](https://user-images.githubusercontent.com/10714292/86154897-8a952380-bab8-11ea-94ba-66098b4ed094.png)